policy-staging.json

```diff
@@ -0,0 +1,22 @@
+{
+  "instance_min_count": 1,
+  "instance_max_count": 3,
+  "scaling_rules": [
+    {
+      "metric_type": "responsetime",
+      "breach_duration_secs": 60,
+      "threshold": 50,
```

Comment:
This is low, but so is all stress on staging.
If this bounces around then 🤷🏻, just proves it is working and can only go between 1 and 3...